packages/web/tailwind.config.js

```diff
@@ -7,20 +7,28 @@ module.exports = {
   theme: {
```

Comment:
@psugihara we should be able to symlink this, no?

Reply:
sure i'll try